doc/getting-started.md

```diff
@@ -82,24 +93,65 @@ most simple capabilities is to add a delay.
 
 ```
 # First, create a rule file
-cat <<EOF > /tmp/hello-5s-rule.yaml
```

Comment:
I hear you.  I had originally planned to include something else.  I switched to this last week because the mixer still doesn't work and this allowed me to create a tutorial with only two pieces.  Adding even a third piece greatly increases the complexity for the new user.  We have Frank's great guide to BookInfo which shows off other kinds of things.

How about we let this PR through, and then work on improving it in follow-ups?